<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
  background-color: #00ffff;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>761</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="Error">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>5</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>成績をCtrl+A(Windows)でコピー＆ペーストしてください．</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
     <widget class="QLabel" name="pic1">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="Send">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>490</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: white;
}</string>
      </property>
      <property name="text">
       <string>単位計算</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>491</width>
        <height>499</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: white;
}</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_l_r">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>教養必修</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_l_r">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_l_re">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>教養選択必修</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_l_re">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_o">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>その他</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_13">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_s_re_k">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>専門 選択必修</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_s_re_k">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_14">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>420</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_s_e_k">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>専門 選択</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_s_e_k">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_15">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>0</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_s_r_b">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>専門 必修 学部</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_s_r_b">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_16">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_s_r_k">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>専門 必修 学科</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_s_r_k">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="pic2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>250</y>
        <width>241</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: white;
}</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_l_des">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>211</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>あああ</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_s_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>175</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_s_des">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>211</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_l_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>211</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_l_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>95</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_s_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_total">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>251</width>
        <height>131</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QPushButton" name="pushButton_l_e">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>221</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget {
  background-color: white;
}</string>
       </property>
       <property name="text">
        <string>教養選択</string>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar_l_e">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>90</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_4</zorder>
     <zorder>groupBox_14</zorder>
     <zorder>groupBox_15</zorder>
     <zorder>groupBox_16</zorder>
     <zorder>pic2</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>groupBox_6</zorder>
     <zorder>groupBox_13</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QTextBrowser" name="state_classes">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>448</width>
        <height>491</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: white;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_prev">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>490</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget {
  background-color: white;
}</string>
      </property>
      <property name="text">
       <string>戻る</string>
      </property>
     </widget>
     <widget class="QLabel" name="pic3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>0</y>
        <width>251</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
